src/mlpack/methods/reinforcement_learning/q_networks/dueling_dqn.hpp

```diff
@@ -0,0 +1,216 @@
+/**
+ * @file methods/reinforcement_learning/q_networks/dueling_dqn.hpp
+ * @author Nishant Kumar
+ *
+ * This file contains the implementation of the dueling deep q network.
```

Comment:
I think we should add a reference to [this](https://arxiv.org/pdf/1511.06581.pdf) paper